app/graphql/albums/resolver.js

```diff
@@ -0,0 +1,26 @@
+const { allAlbums, albumById, photosPerAlbum } = require('../../services/album');
+
+const albumPhotos = id => photosPerAlbum(id);
+
+exports.getAlbum = async (_, params) => {
+  const album = await albumById(params.id);
+  const data = {
+    ...album,
+    artist: album.userId,
+    name: album.title,
+    photos: await photosPerAlbum(params.id)
+  };
+  return data;
+};
+
+exports.getAlbums = async (_, params) => {
+  const tempAlbums = await allAlbums(params.offset, params.limit, params.orderBy);
+  const promisesPhotos = tempAlbums.map(album => albumPhotos(album.id));
```

Comment:
its not necessary to create a new function to do the same. Simply use `photosPerAlbum`.

Also, function name is not descriptive. Consider `photosByAlbumId`, or something like that